### PR TITLE
Pass Disabled prop down in InputText

### PR DESCRIPTION
**Pull request by Mjloturco**
The InputText component currently cannot be disabled, this PR ensures that the disabled prop gets passed through our component down to the html select element.

### CODE REVIEW

**Comment by jaebradley** [comment]
@Mjloturco check out [our commit message syntax](https://github.com/edx/paragon#commit-messages)

**Comment by jaebradley** [comment]
@Mjloturco can you add a story that illustrates the `disabled` prop?

**Comment by jaebradley** [comment]
@Mjloturco do you have commit linting setup correctly? Upper-case scopes should not pass commit message linting

image

**Comment by Mjloturco** [comment]
@jaebradley I will add a story to illustrate and i'll fix the snapshots (though I haven't looked at what those are yet). I may have had a workaround to get that commit through... will rename

File: src/InputSelect/index.jsx [hunk 1]
```diff
@@ -50,6 +50,7 @@ class Select extends React.Component {
         onChange={props.onChange}
         onBlur={props.onBlur}
         ref={props.inputRef}
+        disabled={props.disabled}
```
Comment (jaebradley) [inline src/InputSelect/index.jsx]: Can you add this to [the `Input Select README.md`](https://github.com/edx/paragon/blob/master/src/InputSelect/README.md)

Comment (jaebradley) [reply src/InputSelect/index.jsx]: No you don't need to do this

File: package.json [hunk 2]
```diff
@@ -71,7 +71,7 @@
     "eslint-config-airbnb": "^16.0.0",
     "eslint-config-edx": "^4.0.1",
     "eslint-import-resolver-webpack": "^0.8.1",
-    "eslint-plugin-compat": "^2.1.0",
+    "eslint-plugin-compat": "^2.2.0",
```
Comment (jaebradley) [inline package.json]: Why was this updated?

Comment (jaebradley) [reply package.json]: So it looks like [`eslint-plugin-compat` is pinned to `2.2.0` in `package-lock.json`](https://github.com/edx/paragon/blob/master/package-lock.json#L6213)

image

So this shouldn't change anything.

**Comment by jaebradley** [comment]
alt-text

**Pull request merged**